Makefile

```diff
@@ -21,11 +23,11 @@ apidoc:
 check: luacheck
 
 luacheck:
-	.rocks/bin/luacheck --config .luacheckrc --codes .
+	luacheck --config .luacheckrc --codes .
 
 .PHONY: test
 test:
-	.rocks/bin/luatest -v
+	luatest -v --coverage
 	rm -rf ${CLEANUP_FILES}
 	INDEX_TYPE='TREE' SPACE_TYPE='vinyl' ./test.lua
 	rm -rf ${CLEANUP_FILES}
```

Comment:
side not: do we still needed with rm commands? Probably cleanup logic should live inside our tests.